scripts/find_cord19_length_outliers.py

```diff
@@ -0,0 +1,68 @@
+from pyserini.collection import pycollection
```

Comment:
throw in our standard boilerplate? https://github.com/castorini/pyserini/blob/master/pyserini/setup.py#L1

Reply:
I don't really know how should this import be in the standard boilerplate...